tests/foreman/ui/test_dashboard.py

```diff
@@ -713,6 +714,66 @@ def test_positive_host_collections(self):
                 1
             )
 
+    @skip_if_bug_open('bugzilla', 1417114)
+    @tier2
+    def test_positive_user_access_with_host_filter(self):
+        """Check if user with necessary host permissions can access dashboard
+        and required widgets are rendered
+
+        :id: 24b4b371-cba0-4bc8-bc6a-294c62e0586d
+
+        :Steps:
+
+            1. Specify proper filter with permission for your role
+            2. Create new user and assign role to it
+            3. Login into application using this new user
+            4. Check dashboard and widgets on it
+
+        :Assert: Dashboard and Errata Widget rendered without errors
+
+        :BZ: 1417114
+
+        :CaseLevel: Integration
+        """
+        user_login = gen_string('alpha')
+        user_password = gen_string('alphanumeric')
+        org = entities.Organization().create()
+        # create a role with necessary permissions
+        role = entities.Role().create()
+        entities.Filter(
+            permission=entities.Permission(
+                resource_type='Organization').search(
+                filters={'name': 'view_organizations'}),
```

Comment:
Have you tried `query={'name': 'view_organizations'}` or `query={'search': 'name=view_organizations'}` instead? Filters should be a last resort, as they will grab a list of entities, execute `read()` for each single one and only then form a result list.
tl;dr - filters generate a lot of extra `read()` operations decreasing overall performance, try to avoid them if that's possible.